Lib/test/test_capi/test_misc.py

```diff
@@ -2070,6 +2075,7 @@ def test_configured_settings(self):
 
     @unittest.skipIf(_testsinglephase is None, "test requires _testsinglephase module")
     @unittest.skipUnless(hasattr(os, "pipe"), "requires os.pipe()")
+    @requires_gil_enabled("gh-117649: test does not work in free-threaded build")
```

Comment:
Does it cause a crash or only a failure?  If it doesn't crash than I'd recommend some derivative of `@unittest.expectedFailure` that checks `Py_GIL_DISABLED`.

The problem with skipping outright is that we have to know where to stop skipping later, when we've sorted out the underlying problem.  `unittest.expectedFailure` identifies those places for us.